libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -244,6 +244,11 @@ void GCS_MAVLINK::send_battery_status(const uint8_t instance) const
     } else {
         consumed_wh = -1;
     }
+    int8_t percentage;
```

Comment:
Initialize percentage to -1, and ignore the return of battery.capacity_remaining_pct